module/jobqueue/consumer.go

```diff
@@ -219,6 +219,17 @@ func (c *Consumer) run() (int64, error) {
 	if err != nil {
 		return 0, fmt.Errorf("could not set processed index %v, %w", processedTo, err)
 	}
+
+	for index := c.processedIndex + 1; index <= processedTo; index++ {
+		jobStatus, ok := c.processings[index]
+		if !ok {
+			continue
+		}
+
```

Comment:
```suggestion
```

It's safe to remove a non-existent key.

Reply:
I need to keep it, because I'm accessing [jobStatus.jobID](https://github.com/onflow/flow-go/pull/656/files#diff-2731207c7c63d225021f36118b9379bcb3e4394d9ef95e314020a55ff0715500R230) later. Will crash if `jobStatus` is nil